CHANGELOG.rst

```diff
@@ -3,6 +3,18 @@ Changelog History
 =================
 
 
+xskillscore v0.0.28 (2025-10-15)
+--------------------------------
+
+Bug Fixes
+~~~~~~~~~
+- Fixed a bug introduced in v0.0.27 where `numpy` v1.x support was broken for some
+  algorithms. (:pr:`431`) `Trevor James Smith`_.
```

Comment:
not even sure its a bug

Reply:
That's referring to this commit: https://github.com/xarray-contrib/xskillscore/pull/431/commits/210cf4f891eee7f6a9cb9c179cdee3c897410756

The `trapezoid` function wasn't available for `numpy` v1.0. That was my error.